paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -289,9 +293,15 @@ internal class PaymentSheetActivity : BaseSheetActivity<PaymentSheetResult>() {
             viewModel.checkout(CheckoutIdentifier.SheetBottomGooglePay)
         }
 
-        viewModel.config?.primaryButtonColor?.let {
-            viewBinding.buyButton.backgroundTintList = it
-        }
+        val isDark = baseContext.isSystemDarkTheme()
+        viewBinding.buyButton.backgroundTintList =
+            if (viewModel.config?.primaryButtonColor != null) {
+                viewModel.config?.primaryButtonColor
+            } else {
+                ColorStateList.valueOf(
+                    PaymentsThemeConfig.colors(isDark).primary.toArgb()
+                )
+            }
```

Comment:
This will be intestesting to look at once the entries Colors config is in the PaymentSheet Configuration.  Hopefully this if-else will then go away.